api/IDBVersionChangeEvent.json

```
@@ -68,6 +68,56 @@
           "deprecated": false
         }
       },
+      "IDBVersionChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBVersionChangeEvent/IDBVersionChangeEvent",
+          "spec_url": "https://w3c.github.io/IndexedDB/#dom-idbversionchangeevent-idbversionchangeevent",
+          "description": "<code>IDBVersionChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": true
```

```suggestion
              "version_added": "41"
```

Based on testing https://mdn-bcd-collector.appspot.com/tests/api/IDBVersionChangeEvent/IDBVersionChangeEvent in Chrome 40 and 41

https://github.com/foolip/mdn-bcd-collector#updating-bcd-using-the-results should have been able to give the exact number, but failed here due to https://github.com/foolip/mdn-bcd-collector/issues/1230. @vinyldarkscratch FYI.